configure.ac

```diff
@@ -1,13 +1,14 @@
 AC_PREREQ([2.61])
 dnl Don't forget when updating version number to do GP_CHECK_GEANY also
 AC_INIT([geany-plugins], [2.1])
-AM_INIT_AUTOMAKE([1.8 foreign dist-bzip2 tar-ustar subdir-objects])
-
-m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
 
 AC_CONFIG_SRCDIR([po/POTFILES.in])
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_MACRO_DIR([build/cache])
+AC_CONFIG_AUX_DIR([build/aux])
```

Comment:
This is not strictly related to the PR but I stole it from Geany to reduce the clutter of generated files if using in-source builds.

Reply:
Fair enough, but don't use `aux` as a directory name, it can [break under Windows](https://www.gnu.org/software/autoconf/manual/autoconf-2.60/autoconf.html#File-System-Conventions):

> * Invalid names (lfn)
>   *   Some DOS file names are reserved, and cause problems if you try to use files with those names. These names include `CON`, `AUX`, `COM1`, `COM2`, `COM3`, `COM4`, `LPT1`, `LPT2`, `LPT3`, `NUL`, and `PRN`. File names are case insensitive, so even names like `aux/config.guess` are disallowed.

Reply:
Did I ever mention how much I love Windows?

Anyway, I will rename it to "build/build-aux". A little redundant but still just for generated files.